#! /bin/sh /usr/share/dpatch/dpatch-run
## 30_gtk_wxwidgets_symbol_clash.dpatch by Joachim Breitner <nomeata@debian.org>
##
## All lines beginning with `## DP:' are a description of the patch.
## DP: Avoid symbol clash as it is avoided in http://trac.wxwidgets.org/changeset/61009

@DPATCH@
diff -urNad xaralx-0.7r1785~/wxOil/ftfonts.cpp xaralx-0.7r1785/wxOil/ftfonts.cpp
--- xaralx-0.7r1785~/wxOil/ftfonts.cpp	2007-11-07 01:48:24.000000000 +0100
+++ xaralx-0.7r1785/wxOil/ftfonts.cpp	2009-10-08 10:44:37.000000000 +0200
@@ -122,8 +122,10 @@
 #include FT_TYPE1_TABLES_H
 
 #ifdef __WXGTK20__
+#define GSocket GlibGSocket
 #include <gtk/gtk.h>
 extern GtkWidget *wxGetRootWindow();
+#undef GSocket
 #endif
 
 CC_IMPLEMENT_DYNCREATE( FTFontMan, CCObject )
